src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -363,12 +369,11 @@
   otherwise use the default."
   [config   :- WebserverRawConfig
    num-cpus :- schema/Int]
-  (or (:max-threads config)
-      (let [calc-max-threads (max (calculate-required-threads config num-cpus)
-                                  default-max-threads)]
-        (log/warn "Thread pool size not configured so using a size of "
-                  calc-max-threads)
-        calc-max-threads)))
+  (let [max-threads (or (:max-threads config)
+                        (max (calculate-required-threads config num-cpus)
+                             default-max-threads))]
+    (log/debugf "Using webserver thread pool size of %d" max-threads)
```

Comment:
:+1: the old warning is making me twitchy.

I'd be okay with this being an INFO instead of DEBUG, but, either way is better.